multimodal/setup.py

```diff
@@ -27,13 +27,13 @@
     'boto3',
     'setuptools<=59.5.0',
     'timm<0.6.0',
-    'torch>=1.0,<1.11',
+    'torch>=1.0,<1.12',
     'fairscale>=0.4.5,<0.5.0',
     'scikit-image>=0.19.1,<0.20.0',
     'smart_open>=5.2.1,<5.3.0',
     'pytorch_lightning>=1.5.10,<1.7.0',
     'torchmetrics>=0.7.2,<0.8.0',
-    'transformers>=4.16.2,<4.17.0',
+    'transformers>=4.18.0,<4.21.0',
```

Comment:
Need to upgrade transformers to support Torch 1.11 due to the availability of https://github.com/huggingface/transformers/pull/16043 starting from 4.18.0